packages/platform-server/src/tokens.ts

```diff
@@ -30,6 +30,8 @@ export interface PlatformConfig {
    * Whether to append the absolute URL to any relative HTTP requests. If set to
```

Comment:
nit: we can highlight this a bit more:
```suggestion
   * Note: this option has no effect and can be removed from the config.
   *
   * Whether to append the absolute URL to any relative HTTP requests. If set to
```